tests/config/views.view.content.yml

```diff
@@ -329,16 +329,31 @@ display:
           hide_empty: false
           empty_zero: false
           hide_alter_empty: true
-          destination: true
-          plugin_id: entity_operations
-        moderation_state:
-          id: moderation_state
+          click_sort_column: target_id
```

Comment:
This is the result of a previous PR which moved the operations field. Again, I think we just forgot to merge updated config then. NBD.